pkg/application/componentInstallationUseCase.go

```diff
@@ -3,23 +3,35 @@ package application
 import (
 	"context"
 	"fmt"
-	"sigs.k8s.io/controller-runtime/pkg/log"
-
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain"
 	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain/ecosystem"
 	"github.com/cloudogu/k8s-blueprint-operator/pkg/domainservice"
 	"github.com/cloudogu/k8s-blueprint-operator/pkg/util"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+const (
+	noDowngradesExplanationTextFmt = "downgrades are not allowed as the data model of the %s could have changed and " +
+		"doing rollbacks to older models is not supported. " +
+		"You can downgrade %s by restoring a backup. " +
+		"If you want an 'allow-downgrades' flag, issue a feature request"
+	noDistributionNamespaceSwitchExplanationText = "switching distribution namespace is not allowed. If you want an " +
+		"`allow-switch-distribution-namespace` flag, issue a feature request"
```

Comment:
It should be clear that we're talking about the namespace of components.
Sadly, we called it _distribution namespace_ and _component distribution namespace_ is awfully long.

Reply:
I extend the message with compents context.